tests/e2e/specs/profile/applications-passwords.test.js

```diff
@@ -93,7 +74,11 @@ describe('Manage applications passwords', () => {
 
     it('should correctly revoke all the application passwords', async() => {
         await createApplicationPassword(TEST_APPLICATION_NAME);
-        await revokeAllApplicationPasswords();
+
+        await page.waitForSelector('#revoke-all-application-passwords');
+        await page.click('#revoke-all-application-passwords');
+        await page.keyboard.press('Enter');
```

Comment:
It is not clear why we need this just by reading it in the code. We can explicitly await for the dialog instead.

```js
const revocationDialogPromise = new Promise(resolve => {
  page.once('dialog', resolve);
});

const [dialog] = await Promise.all([
  revocationDialogPromise
  revokeAllApplicationPasswordsButton.click(),
]);

expect(dialog.type()).toBe('confirm');
expect(dialog.message()).toMatchInlineSnapshot();

await dialog.accept();
```

So that we can also make assertions of the text of the confirmation dialog.

Reply:
I just found out that we're using [`enablePageDialogAccept`](https://github.com/WordPress/gutenberg/blob/2d7fd1ba80da3fa9e6af427ba014958b0cd608a8/packages/e2e-tests/config/setup-test-framework.js#L242), which I think is overly aggressive and no way to temporary disable it either. We can omit the final `dialog.accept()` here though I think we should fix it in gutenberg to at least make it configurable.

Reply:
Yep, something like that! Probably worth to still add `await dialog.accept()` but comment it out with reasons I mentioned above.

Also the same should also be done in "revoke all" test below too. 👍

Reply:
Oh and please note that this is more like a nitpick than a requirement. Adding these only matters when we want to assert for the type and the message of the dialog. If we don't care about them, then I think it's also fine to just remove these altogether (including `page.keyboard.press('Enter')`).

Reply:
Yes, I don't think we need to assert of dialog content